src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -810,14 +825,22 @@ private Connection makeNewConnection() throws SQLException {
   private PreparedStatement getDocFromDb(Connection conn,
       String uniqueId) throws SQLException {
     PreparedStatement st = conn.prepareStatement(singleDocContentSql);
-    uniqueKey.setContentSqlValues(st, uniqueId);  
+    uniqueKey.setContentSqlValues(st, uniqueId);
```

Comment:
has something changed on this line?  github shows it as changed.

Reply:
Looks I removed the whitespace at the end. Restored it.